app/test_main.py

```diff
@@ -1,3 +1,46 @@
+from typing import Type
+
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+class TestGetHumanAge:
+    @pytest.mark.parametrize(
+        "cat_age,dog_age,result",
+        [
+            (0, 0, [0, 0]),
+            (14, 14, [0, 0]),
+            (15, 15, [1, 1]),
+            (23, 23, [1, 1]),
+            (24, 24, [2, 2]),
+            (27, 27, [2, 2]),
+            (28, 28, [3, 2]),
+            (100, 100, [21, 17]),
+            (-10, -20, [0, 0]),
+            (10 ** 9, 10 ** 9, [249999996, 199999997])
+        ]
+    )
+    def test_get_human_age_with_correct_values(
+            self,
+            cat_age: int,
+            dog_age: int,
+            result: list
+    ) -> None:
+        assert get_human_age(cat_age, dog_age) == result
+
+    @pytest.mark.parametrize(
+        "cat_age,dog_age,expected_error",
+        [
+            ("value", 12, TypeError),
+            (13, "value", TypeError)
+        ]
+    )
+    def test_get_human_age_with_correct_types_of_value(
```

Comment:
Looks like this test should ne named as something like: "test_get_human_age_with_**in**correct_types_of_value"